client/my-sites/comments/comment-list/index.jsx

```diff
@@ -3,11 +3,18 @@
  */
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { filter, map } from 'lodash';
+import { localize } from 'i18n-calypso';
+import { filter, keyBy, map } from 'lodash';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
```

Comment:
The `ReactCSSTransitionGroup` version we use has explicit timeout props (introduced in https://github.com/facebook/react/pull/4561) that should take care of exactly that issue there.
Considering it's slated to be replaced with `CSSTransitionGroup`, I'd say let's wait for that while we're still behind a feature flag, and then work on a fix if it's really needed.